### PR TITLE
Reorder queen-attack tests

### PR DESCRIPTION
**Pull request by colin-mullikin**
Re-ordered the tests according to issue #241.

### CODE REVIEW

File: exercises/queen-attack/src/test/java/QueenAttackCalculatorTest.java [hunk 1]
```diff
@@ -8,127 +8,127 @@
...
-        assertFalse(calculator.canQueensAttackOneAnother());
+        assertTrue(calculator.canQueensAttackOneAnother());
     }
 
+    /*
+     * See https://github.com/junit-team/junit4/wiki/Rules for information on JUnit Rules in general and
+     * ExpectedExceptions in particular.
+     */
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
```
Comment (FridaTveit) [inline exercises/queen-attack/src/test/java/QueenAttackCalculatorTest.java]: I would keep this at the top of the class, where it was before, even though you're not using it until here. If you keep all field declarations at the top of the class then they're easy to find if you're looking for them :)

Comment (colin-mullikin) [inline exercises/queen-attack/src/test/java/QueenAttackCalculatorTest.java]: Okay. That was the only bit that I was unsure of. I'll move that back to the top where it was.

**Comment by FridaTveit** [comment]
By the way, for future reference it's probably better to make a pull request from a feature branch instead of from your master branch. When you've made a pull request any further changes pushed to the same branch will automatically be added to the pull request. So it's easy to accidentally add changes you didn't intent to :)

**Comment by colin-mullikin** [comment]
Yeah, I realized that immediately after I did it. Figured it would get approved and merged pretty quickly, so I didn't bother to go back and spend the time to do it that way.

**Comment by FridaTveit** [comment]
Fair enough :)

**Pull request merged**